jerry-core/ecma/operations/ecma-proxy-object.c

```diff
@@ -1543,7 +1572,12 @@ ecma_collection_t *
 ecma_proxy_object_own_property_keys (ecma_object_t *obj_p) /**< proxy object */
 {
   JERRY_ASSERT (ECMA_OBJECT_IS_PROXY (obj_p));
-
+/*  if (++JERRY_CONTEXT (proxy_depth) > ECMA_CONFIG_PROXY_RECURSION_DEPTH)
+  {
+    ecma_raise_range_error (ECMA_ERR_MSG ("Maximum call stack size exceeded."));
+    return NULL;
+  }
+*/
```

Comment:
Is this a leftover?

Reply:
ohh yeah was shuffling around the commits (during update) and got here accidentally. Will fix it. I don't know how I've missed this. (and this also applies to the style issue above)